.circleci/config.yml

```diff
@@ -9,21 +9,26 @@ jobs:
       MATPLOTLIBRC: tests
     steps:
       - checkout
+      - run:
+          name: Update to latest `nox`
+          command: |
+            pip uninstall --yes nox-automation
+            pip install "nox >= 2018.10.15"
```

Comment:
This can be removed once the Docker image is updated (I'm just resurfacing an old comment that got "disappeared" when I updated the `nox` version in this line)